examples/deepzoom/deepzoom_tile.py

```diff
@@ -142,8 +147,18 @@ def get_dzi(self):
 class DeepZoomStaticTiler:
     """Handles generation of tiles and metadata for all images in a slide."""
 
-    def __init__(self, slidepath, basename, format, tile_size, overlap,
-                limit_bounds, quality, workers, with_viewer):
+    def __init__(
+        self,
+        slidepath,
+        basename,
+        format,
+        tile_size,
+        overlap,
+        limit_bounds,
+        quality,
+        workers,
+        with_viewer,
```

Comment:
That's... quite unreadable. :frowning:  It doesn't seem this bad in other cases, though, so maybe we just live with this one.

Reply:
It starts to look slightly better when you add type annotations.

```python
def __init__(
    self,
    slidepath: PathLike,
    basename: Filename,
    format: Literal["jpg", "png"],
    tile_size: Tuple[int, int],
    ...
    with_viewer: bool,
):